data_handler/handlers/liquidable_debt/debt_handlers.py

```diff
@@ -49,45 +76,184 @@ def prepare_data(self, protocol_name: str, path: str = LOCAL_STORAGE_PATH) -> di
+                    del result[index][COLLATERAL_FIELD_NAME][collateral_token]
+
+            if not result[index][COLLATERAL_FIELD_NAME].keys():
+                del result[index]
+
+        return result
 
-    def _calculate_liquidable_debt(self, data: dict = None):
+    @staticmethod
+    def get_prices_range(collateral_token_name: str, current_price: Decimal) -> Iterable[Decimal]:
+        """
+        Get prices range based on the current price.
+        :param current_price: Decimal - The current pair price.
+        :param collateral_token_name: str - The name of the collateral token.
+        :return: Iterable[Decimal] - The iterable prices range.
+        """
+        from handlers.helpers import get_range, get_collateral_token_range
+
+        collateral_tokens = ("ETH", "wBTC", "STRK")
+
+        if collateral_token_name in collateral_tokens:
+            return get_collateral_token_range(collateral_token_name, current_price)
+
+        return get_range(Decimal(0), current_price * Decimal("1.3"), Decimal(current_price / 100))
+
+    @classmethod
+    def verify_endpoint_protocol(cls, protocol_name: str) -> None:
+        """
+        Verifies if the protocol exists
+        :param protocol_name: str
+        :return: None
+        """
+        if protocol_name.lower() not in [
+            protocol.value.lower() for protocol in AvailableProtocolID
+        ]:
+            raise ProtocolExistenceError(protocol_name)
+
+    @classmethod
+    def get_interest_rate_models_data(cls, protocol_name: str) -> dict:
+        """
+        Returns an interest rate models data
+        :param protocol_name: str
+        :return: dict
+        """
+        cls.verify_endpoint_protocol(protocol_name)
+        for protocol in AvailableProtocolID:
+            current_protocol = str(protocol.value).lower()
+            if protocol_name.lower() == current_protocol:
+                try:
+                    response = requests.get(cls.INTEREST_MODEL_VALUES_URL.format(protocol=protocol.value))
+                    response.raise_for_status()
+                    return response.json()
+                except:
+                    time.sleep(10)
+                    response = requests.get(cls.INTEREST_MODEL_VALUES_URL.format(protocol=protocol.value))
+                    response.raise_for_status()
+                    return response.json()
+
+    @staticmethod
+    def normalize_protocol_name(protocol_name: str) -> str:
+        """
+        Normalizes the given protocol name.
+        :param protocol_name: str
+        :return: str
+        """
+        return "_".join(protocol_name.split(" ")).lower()
+
+    def _calculate_liquidable_debt(
+            self, data: dict = None,
+    ) -> dict:
         """
         Calculates liquidable debt based on data provided and updates an existing data.
-        :param data: Data to calculate liquidable debt for.
+        Data to calculate liquidable debt for:
+        Args: data( dict[
```

Comment:
Please, rework args docstring for more readable format

Reply:
Done